### PR TITLE
Improve handling of installer dirs

### PR DESCRIPTION
**Pull request by tejohnso**
@fjvallarino @ahmedalsudani I've added integration tests for this and updated the readme to indicate how they are run.  Requesting merge to release-branch.  Please review.  
Note that I have yet to do manual testing on this.  I will test on both win7 and win10.

### CODE REVIEW

**Comment by tejohnso** [comment]
@fjvallarino @ahmedalsudani Ready for merge to release-branch - please review.

**Comment by ahmedalsudani** [comment]
@tejohnso sorry for letting this languish. I'll take a look later today

File: installer/config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var platform = require("rise-common-electron").platform,
 network = require("rise-common-electron").network,
 proxy = require("rise-common-electron").proxy,
 thisInstallerVersion = requireRoot("version.json"),
+electronAppPath = "",
```
Comment (ahmedalsudani) [inline installer/config.js]: @tejohnso is there a reason we can't initialize in the declaration?

i.e. is it possible to move [this call](https://github.com/Rise-Vision/rise-launcher-electron/pull/158/files#diff-7a9076d6d94e62c13d641aa71f19ae8eR30] here?

File: package.json [hunk 2]
```diff
@@ -29,6 +30,8 @@
     "yargs": "3.28.0"
   },
   "scripts": {
-    "test": "./node_modules/jshint/bin/jshint cfg/* && cd external-deps/java-network-test && mvn test && cd ../.. && istanbul cover node_modules/mocha/bin/_mocha -r ./init.js -- --reporter mocha-junit-reporter --reporter-options mochaFile=test/test-results.xml --recursive test/unit"
+    "test": "./node_modules/jshint/bin/jshint cfg/* && cd external-deps/java-network-test && mvn test && cd ../.. && istanbul cover node_modules/mocha/bin/_mocha -r ./init.js -- --reporter mocha-junit-reporter --reporter-options mochaFile=test/test-results.xml --recursive test/unit",
+    "integration": "export ELECTRON_PATH=$PWD/node_modules/electron-prebuilt/dist/electron && node_modules/.bin/electron-mocha -r ./init.js -t 10000 --recursive test/integration"
```
Comment (ahmedalsudani) [inline package.json]: Would this work on Windows? (specifically `$PWD`)

**Comment by tejohnso** [comment]
@ahmedalsudani Let's further discuss the use of the app module across the other modules.  Will merge for now.

**Comment by ahmedalsudani** [comment]
@tejohnso that would be cheating :)

:+1:

**Pull request merged**